android/src/main/java/com/wultra/android/malwarelytics/reactnative/MalwarelyticsNativeHelper.kt

```diff
@@ -0,0 +1,210 @@
+//
+// Copyright 2023 Wultra s.r.o.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions
+// and limitations under the License.
+//
+
+package com.wultra.android.malwarelytics.reactnative
+
+import android.app.Activity
+import android.os.Handler
+import android.os.Looper
+import androidx.annotation.MainThread
+import com.facebook.react.util.RNLog
+import com.wultra.android.appprotection.AppProtection
+import java.util.concurrent.Executors
+
+/**
+ * The `MalwarelyticsNativeHelper` class provides additional functionality for application using Malwarelytics from
+ * the native code.
+ */
+class MalwarelyticsNativeHelper private constructor(
+    /**
+     * Instance of AppProtection
+     */
+    internal val appProtectionService: AppProtection
+) {
+
+    companion object {
+        /**
+         * Shared instance of the helper class.
+         */
+        @JvmStatic
+        val instance: MalwarelyticsNativeHelper by lazy {
+            MalwarelyticsNativeHelper(AppProtection.getInstance())
+        }
+    }
+
+    /**
+     * Contains `true` if service is initialized.
+     */
+    internal val isServiceInitialized: Boolean
+        get() = appProtectionService.isInitialized()
+
+    /**
+     * Contains all activities registered for tapjacking protection.
+     */
+    private val activitiesRegisteredForTapjackingProtection = mutableListOf<Activity>()
+
+    /**
+     * Register activity for protection against the tapjacking. This method is useful in case that React-Application
+     * has its own activities created and used before the Malwarelytics module is initialized fom the JavaScript.
+     * You don't need to use this method once the module is initialized. All future activities will be protected
+     * automatically.
+     *
+     * @param activity Activity to protect.
+     */
+    @MainThread
+    fun registerActivityForTapjackingProtection(activity: Activity) {
```

Comment:
Bad name. Better keep it as something like `registerActivityForProtection(Activity)`. 
SDK function`raspManager.protectActivity(Activity)` handles more features than tapjacking. It also handles screen reader protection and screenshot protection.